package.json

```diff
@@ -94,6 +95,7 @@
     "unmock-fetch": "file:packages/unmock-fetch",
     "unmock-jest": "file:packages/unmock-jest",
     "unmock-node": "file:packages/unmock-node",
+    "unmock-server": "file:packages/unmock-server",
```

Comment:
Should this also be included in the `lint-ts` and `lint-ts-fix` scripts above?

Reply:
👍